net-misc/passt/Manifest

```diff
@@ -2,3 +2,4 @@ DIST passt-2023.12.04.tar.xz 188136 BLAKE2B f1d72fbb4bbb25f6a11e9ff87a91cf9bac06
 DIST passt-2023.12.30.tar.xz 188596 BLAKE2B 75712a7cb7a00fea20427979153090d938b8c7b0e3da61c83cabb3751786034df35ebbe8afbd991d582a680f55b12792547b68efa931a5576dc181040b31e5fd SHA512 4b19d191e2526bacc61bb8ab2964f5da4e0535656b094a7592f91d6b437c855a586fc772e2abd89138ff4f89aae12ade494c6e0fe7bee1502fe319215200d8f0
 DIST passt-2024.03.20.tar.xz 195332 BLAKE2B 1bed740ddd235f300a4233fb2024f155758809589f1fdd2174ab99a8a93dd16044ab41814b215d12eb1e557966a73a3754bbff2b9acd752b45b385edd88ba3c7 SHA512 3701820e539545771d0f867fe0b8977cb5967e7501413f717fed08d2d2c141f17d86ed7238177b4fb4cf4cfc01c6c9d07e3c218c34b48e0a5a1f2b2ce8a5b903
 DIST passt-2024.03.26.tar.xz 195348 BLAKE2B d91e6ac1d50aae04ceb53a34341b545ff644ceccdfcfab461907050612d9d871f61591aac02ebddef9a607a9b39ce5017a2fc47c9189f85b9b9a70a30d010e4c SHA512 da771f8f334862f81702b22688c9d3552aefc2883b5a44c39156ff7a69f301cabf52fa0fb13f8695240db5411c498806d28d847fca8caff2bedb69d1a4b4e5a7
+DIST passt-2024.04.05.tar.lz 197125 BLAKE2B 2d3b1f9be505b235001e22fc4eb7bb782c89bc6383993a2f1c80223f1d94d9bb84ddfa6235bb371b9ead2df9d36d62b60ce37b9f15b7fda5633353e459f34233 SHA512 f023ba433ebf54b46b345f9692418b6140fdf080a65fe1a1b1332cc8819645ca012088f4d3658886b52aa2dcb734fc1d4594e951f57cf27d28ecc55e538d7c4e
```

Comment:
Sorry, no, let's not do this kneejerk thing.

Reply:
I initially chose zstd as I prefer it. But lz is 47 KiB smaller so finalized on it. 
If you insist I'll revert back to xz.

Reply:
I do insist. Neither zstd and lz are supported by PMS out of the box so you have to use unpacker.eclass for no real benefit. Just use .xz.